tests/v1alpha_slo_test.go

```diff
@@ -293,5 +293,8 @@ func assertV1alphaSLOsAreEqual(t *testing.T, expected, actual v1alphaSLO.SLO) {
 	actual.Spec.CreatedBy = ""
 	actual.Status = nil
 	actual.Spec.TimeWindows[0].Period = nil
+	if actual.Spec.Objectives[0].Value == nil {
+		actual.Spec.Objectives[0].Value = ptr(0.0)
+	}
```

Comment:
Shouldn't this be only set when the SLO is composite SLO?

Reply:
or value will be always only `nil` if the SLO is a composite SLO?

Reply:
Could: yes, should: maybe, must: no. This change needs to live only for a few days, so I just didn't bother. But... you're the reviewer here, so I'll leave it to your decision.

Reply:
Before changes to N9, `value` is never `nil`. After changes to N9, `value` can only be `nil` for Composite SLOs which will be tested in https://github.com/nobl9/nobl9-go/pull/549. Value can be not `nil` if Composite SLO was created prior to change and had non-zero value.

Reply:
> After changes to N9, value can only be nil for Composite SLOs
that's good enough for me :)